shell/browser/native_window_views.cc

```diff
@@ -643,6 +644,31 @@ void NativeWindowViews::SetResizable(bool resizable) {
   resizable_ = resizable;
 }
 
+bool NativeWindowViews::MoveAbove(const std::string& sourceId) {
+  const content::DesktopMediaID id = content::DesktopMediaID::Parse(sourceId);
+  if (id.type != content::DesktopMediaID::TYPE_WINDOW)
+    return false;
+
+#if defined(OS_WIN)
+  const HWND otherWindow = reinterpret_cast<HWND>(id.id);
+  if (!IsWindow(otherWindow))
```

Comment:
Add `::`prefix to make it clear that this is a win32 API call. 

```suggestion
  if (!::IsWindow(otherWindow))
```

Reply:
Done, thx!